### PR TITLE
add config.floatSpaceAlwaysShow to always show inline toolbar

### PR DESCRIPTION
**Pull request by caffodian**

### CODE REVIEW

File: tests/plugins/floatingspace/alwaysshow.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+bender.test( {
+	'test floatingSpaceLayout event': function() {
+		var editor = this.editor;
+		var floatingSpace = editor.document.getById('cke_' + editor.name);
+
+		editor.focus();
+
+		assert.areSame(floatingSpace.getStyle('display'), "");
+
+		editor.fire( 'blur' );
```
Comment (caffodian) [inline tests/plugins/floatingspace/alwaysshow.js:25]: manually fire the blur (un-focus) event on the editor

File: tests/plugins/floatingspace/alwaysshow.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+		assert.areSame(floatingSpace.getStyle('display'), "");
```
Comment (caffodian) [inline tests/plugins/floatingspace/alwaysshow.js:26]: previously it would set display to "none"

File: tests/plugins/floatingspace/alwaysshow.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	}
+} );
```
Comment (bocribbz) [inline tests/plugins/floatingspace/alwaysshow.js:28]: Wow cool.
I think this is the first javascript tests i have ever seen. 👍

**Comment by bocribbz** [comment]
Ping me when you have the beta test plan.
Also a screenshot would be helpful too..

**Comment by caffodian** [comment]
The beta test plan will go into https://github.com/PolicyStat/PolicyStat/issues/2734 when it's ready in a bit - this is just for the JS side of things.  @bocribbz 

screenshot isn't super exciting, it's basically the old view, except the floating toolbar stays visible, even when not focused (you can tell by the lack of html block outlines + the highlight thingy) 

screen shot 2016-04-25 at 1 15 05 pm

**Comment by bocribbz** [comment]
Looks good 👍

**Pull request merged**